packages/data-point-fetch/README.md

```diff
@@ -0,0 +1,38 @@
+# @data-point/fetch
+
+> DataPoint Reducer that wraps the node-fetch module.
+
+**Prerequisites:**
+
+Node v10 LTS or higher.
+
+**Installing:**
+
+```bash
+npm install --save @data-point/fetch
+```
+
+For documentation on this module's api: [http://data-point.github.com/reducers/fetch](http://data-point.github.com/reducers/fetch).
+
+For data-point's documentation visit its [home page](http://data-point.github.com) ([http://data-point.github.com](http://data-point.github.com)).
```

Comment:
this and the url from above are wrong, will update them once we know their final destination.